zerver/openapi/markdown_extension.py

```diff
@@ -76,6 +76,13 @@
     "string": "demo",
     "boolean": False,
 }
+ADMIN_CONFIG_FUNCTIONS = [
+    "/realm/profile_fields:post",
+    "/users:post",
+    "/messages/{message_id}:delete",
+    "/realm/profile_fields:patch",
+]
```

Comment:
I think structurally, whether an endpoint requires an organization administrator to use it will be a property of the endpoint, and thus should probably live in the OpenAPI files.  This is an OK intermediate state, but maybe we should add an `x-requires-administrator` tag using https://swagger.io/docs/specification/openapi-extensions/?

We will likely also want to have that flag control whether to show a "requires organization administrator" banner at the top of the page like we do in many Help Center articles.